src/components/annotations3d/attributes.js

```diff
@@ -72,9 +72,13 @@ module.exports = overrideAll({
     showarrow: annAtts.showarrow,
     arrowcolor: annAtts.arrowcolor,
     arrowhead: annAtts.arrowhead,
+    startarrowhead: annAtts.startarrowhead,
+    arrowside: annAtts.arrowside,
     arrowsize: annAtts.arrowsize,
+    startarrowsize: annAtts.startarrowsize,
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
+    startstandoff: annAtts.startstandoff,
```

Comment:
Does this really work in 3D or are these attributes there just to make the supply-default function not error out?

Reply:
Yes, it works fine

Reply:
Great. Would you mind adding an annotations item showing off the new attributes to [`gl3d_annotations.json`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/gl3d_annotations.json)?

Reply:
Done